statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -1209,6 +1402,92 @@ def forecast(self, steps=1, **kwargs):
         """
         return self.predict(start=self.nobs, end=self.nobs+steps-1, **kwargs)
 
+    def plot_diagnostics(self, variable=0, lags=10, fig=None, figsize=None):
+        """
+        Diagnostic plots for the standardized residual assocaited with one
```

Comment:
typo `associated`

description/summary line should be one line, otherwise we might have problems with sphinx
maybe
`Diagnostic plots for the standardized residual of one endogenous variable`